test/api.spec.js

```diff
@@ -0,0 +1,389 @@
+import { api } from '../src/api';
+
+const options = {
+  userId: 'user123456',
+  apiKey: 'sample',
+  secretKey: 'drH5zDpDMLLA6elOPVl7tQ==',
+  itemType: 'product',
+  listId: 'c15ca47b-a980-5ba9-af14-ad38a6b043d6',
+};
+
+describe('Api', function() {
+  describe('getList', function() {
+    beforeEach(function() {
+      this.server = sinon.createFakeServer();
```

Comment:
não precisa usar o this, basta colocar em uma const fora do beforeEach e usar ele do jeito que vc quer